modules/setting-up-cpu-manager.adoc

```diff
@@ -172,24 +179,45 @@ QoS Class:       Guaranteed
 Node-Selectors:  cpumanager=true
 ----
 
-. Verify that the `cgroups` are set up correctly. Get the process ID (PID) of the `pause` process:
+. Verify that a CPU has been exclusively assigned to the pod by running the following command:
 +
 [source,terminal]
 ----
+# oc describe node --selector='cpumanager=true' | grep -i cpumanager- -B2
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE    NAME                CPU Requests  CPU Limits  Memory Requests  Memory Limits  Age
+cpuman       cpumanager-mlrrz    1 (28%)       1 (28%)     1G (13%)         1G (13%)       27m
+----
+
+. Verify that the `cgroups` are set up correctly. Get the process ID (PID) of the `pause` process by running the following command:
++
+[source,terminal]
+----
+# oc debug node/perf-node.example.com
+sh-4.2# systemctl status | grep -B5 pause
+----
```

Comment:
@prb112 @madeelibm Is it correct like this?

Reply:
I think it's a little cleaner to use `crictl inspect $(crictl ps --output json | jq -r '.[] | .[] | select(.metadata.name == "pause").id' | tail -n 1) | jq -r '.info.pid'`

Reply:
The command doesn't work even when i run with sudo:
```
[core@worker-1 ~]$ sudo crictl inspect $(crictl ps --output json | jq -r '.[] | .[] | select(.metadata.name == "pause").id' | tail -n 1) | jq -r '.info.pid'
FATA[0000] validate service connection: validate CRI v1 runtime API for endpoint "unix:///var/run/crio/crio.sock": rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial unix /var/run/crio/crio.sock: connect: permission denied"
FATA[0000] ID cannot be empty
```

Reply:
Also we need an output like mentioned in the next para. 
`           │ └─kubepods-pod5b51326a_f1e6_4d22_93cc_1b79570ab5cb.slice
           │   ├─crio-c91e8b8df26758821d701aa4f1f02134becf9086edf3b887bb8221eb24ca04ef.scope
           │   │ └─5548 /pause`

Reply:
yeah - I'm not in love with the grep solution. It's unreliable. 

```
sudo crictl inspect $(crictl ps --output json | jq -r '.[] | .[] | select(.metadata.name == "pause").id' | tail -n 1) | jq -r '.info.pid'
```
If it's not working for you, it means you probably don't have a pause container. It also might be the difference between systems.

Reply:
pause process was running with PID 5448 as mentioned above. 
Grep is also used in other steps. I don't know may be we can leave the command and just write get the PID of the pause process as in the original doc.

Reply:
Well... it's reasonable to use `grep` however, if there are two processes, the above is very misleading with -B5. Maybe just add a warning that you'll have to distinguish two pause containers if they are used.